src/locales/en/common.json

```diff
@@ -993,7 +993,12 @@
     "synchronizing": "Synchronizing",
     "synchronizingDesc": "We are synchronizing your accounts, this can take a while...",
     "noAccountToCreate": "No <1><0>{{currencyName}}</></> account was found to create. Restart the flow and sync your accounts",
-    "cantCreateAccount": "There are no transactions on your last created <1><0>{{accountName}}</></> account. You must first receive crypto assets on that account before you can add a new one."
+    "cantCreateAccount": "There are no transactions on your last created <1><0>{{accountName}}</></> account. You must first receive crypto assets on that account before you can add a new one.",
+    "tokens": {
+      "title": "Add Token",
+      "disclaimer": "{{tokenName}} is a ERC20 Token.\nYou can receive tokens directly on a Ethereum account",
```

Comment:
_"**an** ERC20 Token" perhaps?
"**an** Ethereum account"_
@dasilvarosa

Reply:
I thought so too but I used the text from Zeplin

Reply:
🤷‍♂

Reply:
Agree with @juan-corted:

"an ERC20 Token" 
"an Ethereum account"